pyclibrary/backends/ctypes.py

```diff
@@ -295,14 +295,13 @@ def _get_pointer(self, arg_type, sig):
         """Build an uninitialised pointer for the given type.
 
         """
-        if (arg_type == ['void', '**'] or
-                arg_type == ['void', '*', '*']):
+        if arg_type == ['void', '*', '*']:
             cls = c_void_p
         else:
             # Must be 2-part type, second part must be '*' or '**'
-            assert len(arg_type) == 2 and arg_type[1] in ('*', '**')
+            assert 2 <= len(arg_type) <= 3 and set(arg_type[1:]) == {'*'}
             cls = self._get_type(sig, pointers=False)
-            if arg_type[1] == '*':
+            if arg_type[1:] == ('*',):
                 return pointer(cls(0))
             else:
```

Comment:
This won't work if there is a pointer to pointer to pointer we would need a loop here.